core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -431,9 +437,8 @@ private void closeBlockInStream(BlockInStream stream) throws IOException {
         return;
       }
       // trigger async caching
-      boolean nettyTransEnabled =
-          Configuration.global().getBoolean(PropertyKey.USER_NETTY_DATA_TRANSMISSION_ENABLED);
-      if (nettyTransEnabled) {
+      // dora don't need to trigger async caching as the PagedFileReader will cache it after reading
+      if (mNettyTransEnabled && !mDoraEnabled) {
         triggerNettyAsyncCaching(stream);
```

Comment:
Can we use gRPC to send the cache request, even if Netty is enabled?

Reply:
Good idea. We can leave this as a `TODO` action: the client end call gRPC to notify the worker to cache the file, and then the worker cache the file with netty or gRPC according to whether netty is enable or not.